server/proxy/pfreerdp.c

```diff
@@ -19,11 +19,41 @@
  */
 
 #include "pf_server.h"
+#include "proxy.h"
+#include "pf_config.h"
+#include "pf_log.h"
 
 int main(int argc, char* argv[])
 {
-	char* host = "0.0.0.0";
-	long port = 3389;
-	BOOL localOnly = FALSE;
-	return pf_server_start(host, port, localOnly);
+	int status = 0;
+	rdpProxyServer* server;
+	server = proxy_server_new();
+
+	if (server == NULL)
+	{
+		status = -1;
+		goto fail;
+	}
+
+	if (!pf_server_load_config("config.ini", server->config))
+	{
+		WLog_ERR(TAG, "An error occured while parsing configuration file");
+		status = -1;
+		goto fail;
+	}
+
+	WLog_INFO(TAG, "Loaded server, allowing channels:");
+
+	for (int i = 0; i < server->config->AllowedChannelsCount; i++)
+		printf("  - %s\n", server->config->AllowedChannels[i]);
```

Comment:
IDK if a printf is a good way to do it, I understand you don't want the TAG but you can't disable a printf